### ISSUE TITLE
Submit value

### ISSUE THREAD

**Issue by DONSA** [bootsz/wp-advanced-search#123]
Hi,
I can't change the label in submit button to "Pesquisar":

`$args['fields'][] = array(
        'type' => 'submit',
        'class' => 'btn',
        'value' => 'Pesquisar'
);`

**Comment by bibi-babi** [comment]
I use this, and  no problem with action button: 
$args['fields'][] = array('type' => 'submit',
                                    'value' => __('Avvia Ricerca','languagetheme-custom-post'),
                             );

"__('Avvia Ricerca','languagetheme-custom-post')" for multilanguage

**Comment by DONSA** [comment]
Where should I insert the `"__('Avvia Ricerca','languagetheme-custom-post')" for multilanguage` code.

**Comment by bibi-babi** [comment]
> $args['fields'][] = array(
> 'type' => 'submit',
> 'value' => __(Start Search','languagetheme-custom-post'),
> );



Copy the code above identical

##### explanation elements

" Start Search" is the word that you'd like , you can put like this: "do research " , " active filters " " refines the search ." What you like .

" languagetheme -custom - post " is the theme of textdomain . Activated by a function " load_theme_textdomain " ( search the site to understand how to use it ) .

" ' type ' = > ' submit' ' is obviously to tell the system that you're giving the name of the search button .

the "class " of the div that will contain this button is the default " wpas - submit" . You can choose to set the class or you use this and change it in the css .

**Comment by DONSA** [comment]
Thanks but is still not working. I even tried to remove the type as you suggest.

**Comment by bibi-babi** [comment]
remove the string of code "type" does not solve anything. Propro fact is this code that tells the program that are the settings for the search button.

But exactly what is your purpose?
I understood that you wanted to change the name Seach with Pesquisar.
Right?
And it should work fine.
At best, try to replace

```  'value' => __ ('Pesquisar', 'languagetheme-custom-post'),  ```


with

```   'value' => 'Pesquisar',   ```

so even if you do not see the button with the name you wanted.
Then the problem are in this specific function but in the system.
In short, you have a bug somewhere else

**Comment by DONSA** [comment]
The string does't show in WPML to translate.
I tried to replace to
`'value' => 'Pesquisar'` but didn't work. The text in the button is always "Search".

**Comment by bibi-babi** [comment]
sorry, but at this point I think you're modifying the code of the page you're viewing.
Because if nothing changes, nothing of what you change, then the changes will not come to be communicated.
The reasons? many: you're on another page php, ftp upload is not working properly. you have some cache system on site that makes you see the old codes.

Comepletamente tried to clean the browser cache (including cookies otherwise there is no use)?

I think the problem is not to this set of search codes. but in your system ...

**Comment by bibi-babi** [comment]
I use WPML, but the code we discuss above I translated it with a language file within the theme.
Not in certro transferred to WPML ...

In addition you may also try to disable WPML and see if he is the problem. You think?

You ask assistance for the development of this code, when in fact the problem is with the interaction with other plugins? and write only the fifth post?

It's not that I have plenty of time to devote to you. By the way I use this code, not the programmer.

**Comment by bootsz** [comment]
Are you using the beta release?  It looks like there is a bug in that version.  I will try to get it fixed soon.

**Comment by bibi-babi** [comment]
normal version this command works fine.
this you can confirm it that I use currently.

**Comment by DONSA** [comment]
Yes, I'm using the beta release because I needed to search a value between a range.
I will what for the fix.
Thank you both for your time.

**Comment by bootsz** [comment]
Try downloading the code again now, it should be fixed

**Comment by DONSA** [comment]
Fix the problem. Thanks.
